.github/workflows/publish.yml

```diff
@@ -0,0 +1,59 @@
+name: 'Publish'
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+env:
+  VERSION: ${{ github.ref_name }}
+  CC_CLEVER_TOOLS_RELEASES_CELLAR_KEY_ID: ${{ secrets.CC_CLEVER_TOOLS_RELEASES_CELLAR_KEY_ID }}
+  CC_CLEVER_TOOLS_RELEASES_CELLAR_SECRET_KEY: ${{ secrets.CC_CLEVER_TOOLS_RELEASES_CELLAR_SECRET_KEY }}
+  RPM_GPG_PRIVATE_KEY: ${{ secrets.RPM_GPG_PRIVATE_KEY }}
+  RPM_GPG_NAME: 'Clever Cloud Nexus (rpm)'
+  RPM_GPG_PASS: ${{ secrets.RPM_GPG_PASSPHRASE }}
+  NEXUS_USER: ${{ secrets.NEXUS_USER }}
+  NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+jobs:
+  publish_cellar:
+    name: 'Publish to Cellar'
```

Comment:
I only see an upload to Github ?

Reply:
The name of this step is wrong because it publishes on every target (see line 51).